torch/functional.py

```diff
@@ -286,7 +286,7 @@ def stft(input, n_fft, hop_length=None, win_length=None, window=None,
 
     .. math::
         X[m, \omega] = \sum_{k = 0}^{\text{win\_length}}%
-                            window[k]\ input[m \times hop_length + k]\ %
+                            window[k]\ input[m \times \text{hop\_length} + k]\ %
                             e^{- j \frac{2 \pi \cdot \omega k}{\text{win\_length}}},
```

Comment:
not a big fan of `frac` after `e^` because it is  hard to read text at `frac`, maybe use `exp{}` instead?